.github/workflows/docker-publish.yml

```diff
@@ -54,22 +51,32 @@ jobs:
         run: |
           KIND_VERSION=$(cat KIND_VERSION)
           BASE_IMAGE=$(cat BASE_IMAGE)
+          KUBERNETES_VERSION=$(curl -Ls https://dl.k8s.io/release/stable.txt)
           echo "KIND_VERSION=$KIND_VERSION" >> $GITHUB_ENV
           echo "BASE_IMAGE=$BASE_IMAGE" >> $GITHUB_ENV
+          echo "KUBERNETES_VERSION=$KUBERNETES_VERSION" >> $GITHUB_ENV
 
-      - name: Install kind
+      - name: Build and install kind - ${{ env.KIND_VERSION }}
         run: |
-          curl -Lo ./kind https://kind.sigs.k8s.io/dl/${{ env.KIND_VERSION }}/kind-linux-amd64
-          chmod +x ./kind
-          sudo mv ./kind /usr/local/bin/kind
+          # The runner has kind pre-installed, removing the same to download the preferred version.
+          rm -rf $(which kind)
+          echo $GOPATH
+          git clone --depth 1 --branch ${{ env.KIND_VERSION }} https://github.com/kubernetes-sigs/kind.git
+          pushd kind
+          git apply ../build-ppc64le.patch 
+          popd
+          make -C kind install
+          mv /home/runner/go/bin/kind /usr/local/bin/
```

Comment:
wondering if we can use extend the path to use the binary from the gobin instead of moving the binary to the usr bin dir!

Reply:
The environment doesn't seem to have the GOPATH/bin added to PATH.
Will include `export PATH=$PATH:$(go env GOPATH)/bin` in `run`